src/server/verification/verificationAPI.js

```diff
@@ -170,7 +170,7 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
           if (claimQueueAllowed > 0 && false === isApprovedToClaim) {
             throw new Error('User not approved to claim, not in queue or still pending')
           }
-
+          req.checkConnection() //check if user request aborted
```

Comment:
```suggestion
          req.assertConnection() // throws if request aborted
```

Reply:
also wrap res.json() calls to  `if (!req.aborted) {}`:

(line 180):

```javascript

if (!req.aborted) {
  res.status(400).json({ success: false, error: message })
}
```

(line 184)
```javascript

if (!req.aborted) {
  res.json(enrollmentResult)
}
```